keadm/app/cmd/util/common.go

```diff
@@ -32,6 +32,8 @@ import (
 
 //Constants used by installers
 const (
+	InstallersOnline = true
```

Comment:
How to change the mode mow? :)

Reply:
patch source ? this is what i am doing with by my debian packaging branch maybe I could add an --offline flag ?